docs/docs/configuration/offline-readers.md

```diff
@@ -0,0 +1,12 @@
+# Offline Readers
+
+This feature lets you display offline readers of topics. If your topics are read in external way, you can provide implementation that will be used to show this information in Console.
```

Comment:
Let's give an example to make things easier to understand:

... external way (e.g. messages getting transferred to Hadoop and analysed by users there), you can ...